pimd/pim_ifchannel.c

```diff
@@ -589,7 +599,7 @@ static int on_ifjoin_expiry_timer(struct thread *t)
 
   ch->t_ifjoin_expiry_timer = NULL;
 
-  ifjoin_to_noinfo(ch);
+  ifjoin_to_noinfo(ch, 1);
```

Comment:
Hello Jafar,
I have changed to bool value rather uni8_t for now. This would make clear objective of true/false. 
Regards,
Chirag